python/cugraph/graph/c_graph.pyx

```diff
@@ -358,38 +343,45 @@ class Graph:
         the existing graph.
         """
         cdef uintptr_t graph = self.graph_ptr
-        err = gdf_add_transposed_adj_list(< gdf_graph *> graph)
+        err = gdf_add_transposed_adj_list(<gdf_graph*> graph)
         cudf.bindings.cudf_cpp.check_gdf_error(err)
 
     def view_transposed_adj_list(self):
         """
         Display the transposed adjacency list. Compute it if needed.
         """
         cdef uintptr_t graph = self.graph_ptr
-        cdef gdf_graph * g = < gdf_graph *> graph
+        cdef gdf_graph * g = <gdf_graph*> graph
         err = gdf_add_transposed_adj_list(g)
         cudf.bindings.cudf_cpp.check_gdf_error(err)
 
-        off_size = g.transposedAdjList.offsets.size
-        ind_size = g.transposedAdjList.indices.size
+        offset_col_size = self.num_vertices() + 1
```

Comment:
Same here, getting the size of the column directly is simpler.

Reply:
I disagree for the same reason.